src/MeMoModel.py

```diff
@@ -67,24 +73,45 @@ def fromSBML(cls, model: sbml.Model) -> MeMoModel:
         _id = model.getId()
         return MeMoModel(metabolites=metabolites, _id=_id)
 
-    def annotate(self) -> None:
+    def annotate(self) -> AnnotationResult:
         """Goes through the different bulk annotation methods and tries to annotate InChI strings to the metabolites
         in the model"""
-        # count the number of newly annotated metabolites
-        anno_result= AnnotationResult(0,0,0)
-        # BiGG
-        temp_result = annotateBiGG(self.metabolites)
-        print("BiGG:",temp_result)
-        anno_result = anno_result + temp_result
-        # Use ChEBI
-        temp = annotateChEBI(self.metabolites)
-        print("ChEBI:",temp_result)
-        anno_result = anno_result + temp_result
-        # GO BULK WISE ThORUGH BIGG AND VMH AND MODELSEED, try to extract as much as possible
-        temp_result = annotateModelSEED(self.metabolites)
-        print("ModelSEED:", temp_result)
-        anno_result = anno_result + temp_result
-        print("Total:", anno_result)
+        print(self._id)
+        
+        #TEMP FIX
+        annotationDict = {"VMH" : annotateVMH_id,
+                          "ModelSEED": annotateModelSEED_id,
+                          "BiGG": annotateBiGG_id
+                          }
+
+        origin_dbs = origin_databases(self.metabolites)
+        origin_db = max(origin_dbs, key = lambda k: origin_dbs[k])
+        print(f"ORIG DB {origin_db}")
+        
+        logger.debug(origin_db)
+         # count the number of newly annotated metabolites
+        anno_result = annotationDict[origin_db](self.metabolites)
+        while True:
+          old_res = AnnotationResult.fromAnnotation(anno_result)
+          # BiGG
+          temp_result = annotateBiGG(self.metabolites)
+          #print("BiGG:",temp_result)
+          anno_result = anno_result + temp_result
+          # Use ChEBI
+          temp_result = annotateChEBI(self.metabolites)
+          #print("ChEBI:",temp_result)
+          anno_result = anno_result + temp_result
+          # GO BULK WISE ThORUGH BIGG AND VMH AND MODELSEED, try to extract as much as possible
+          temp_result = annotateModelSEED(self.metabolites)
```

Comment:
Looks like here is a `anno_result = anno_result + temp_result` missing after ModelSeed annotation. Please confirm!